sony/displays/bravia_rest_spec.cr

```diff
@@ -0,0 +1,419 @@
+require "placeos-driver/spec"
+
+DriverSpecs.mock_driver "Sony::Displays::BraviaRest" do
+  settings({
+    psk: "test1234",
+  })
+
+  # Power Control Tests
+  it "should power on the display" do
```

Comment:
These `it` methods won't work here, but I'll update the driver spec runner so it works as expected